### PR TITLE
One line per sentence in paragraphs

### PR DESCRIPTION
**Pull request by fabienrohrer**
For paragraphs only: switch the remaining lines having the 80 characters maximum constraint to a one sentence by line format.

### CODE REVIEW

**Comment by fabienrohrer** [comment]
Note that lists (items or numbers), notes and tables are not taken into account in this PR (it will be simpler to do this in another PR).

**Comment by fabienrohrer** [comment]
👍

**Comment by fabienrohrer** [comment]
I'm fixing bugs with the `e.g.` statement.

File: guide/introduction-to-webots.md [hunk 1]
```diff
@@ -2,28 +2,18 @@
...
-to a real robot like e-puck, DARwIn-OP, Nao, etc. Adding new interfaces is
-possible through the related sytem.
+Webots is a professional mobile robot simulation software package.
+It offers a rapid prototyping environment, that allows the user to create 3D virtual worlds with physics properties such as mass, joints, friction coefficients, etc. The user can add simple passive objects or active objects called mobile robots.
+These robots can have different locomotion schemes (wheeled robots, legged robots, or flying robots).
+Moreover, they may be equipped with a number of sensor and actuator devices, such as distance sensors, drive wheels, cameras, motors, touch sensors, emitters, receivers, etc. Finally, the user can program each robot individually to exhibit the desired behavior.
+Webots contains a large number of robot models and controller program examples to help users get started.
+
+Webots also contains a number of interfaces to real mobile robots, so that once your simulated robot behaves as expected, you can transfer its control program to a real robot like e-puck, DARwIn-OP, Nao, etc. Adding new interfaces is possible through the related sytem.
```
Comment (DavidMansolino) [inline guide/introduction-to-webots.md]: In this kind of case a new line is probably wanted after the `etc.`

File: guide/modeling.md [hunk 2]
```diff
@@ -79,11 +75,9 @@ and the value of `encoderNoise` must be reset to -1
 
 ### How can I create a passive joint?
 
-First of all, any joint, passive or active, must be created by adding a
-`Joint`-derived node (depending on the constraint type requested) in the Scene
-Tree. A `Joint` is passive if its device is null (or at least not a
-`Motor`-derived node. Alternatively, it is also possible to make a `Motor`
-become passive during the simulation; this can be done like this:
+First of all, any joint, passive or active, must be created by adding a `Joint`-derived node (depending on the constraint type requested) in the Scene Tree.
+A `Joint` is passive if its device is null (or at least not a `Motor`-derived node.
```
Comment (DavidMansolino) [inline guide/modeling.md]: Not related to this PR but his parenthesis is not closed.

**Comment by DavidMansolino** [comment]
I checked ~30 randomly selected files and except the small problem with `etc.` everything looks perfect 😃

**Comment by fabienrohrer** [comment]
@DavidMansolino I took your comments into consideration and found and fix a new bunch of small issues.

👍

File: tests/test_paragraphs.py [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+    def test_one_sentence_per_line(self):
+        """Test that each sentence is written on one line."""
+        for p in self.paragraphs:
+            lines = p['paragraph'].split('\n')
+            for line in lines:
+                line = line.strip()
+                if len(line) == 0:
+                    continue
+                if '**Keywords**' in line:
```
Comment (DavidMansolino) [inline tests/test_paragraphs.py]: Would be nice to write a small comment explaining why we skip this line.

Comment (fabienrohrer) [reply tests/test_paragraphs.py]: Even simpler, this is no more required. Indeed, some special lines are acting like titles and so it's not required to test that these titles are ending with points, etc. But this is now filtered out correctly in the test setup.

**Comment by fabienrohrer** [comment]
👍

**Pull request merged**